doc/admin_doc/install_doc/installation.rst

```diff
@@ -127,13 +185,22 @@ The next step is to install `Docker for Mac <https://docs.docker.com/docker-for-
 Installing INGInious
 --------------------
 
-The recommended setup is to install INGInious via pip and the master branch of the INGInious git repository.
-This allows you to use the latest development version. This version is currently the supported one for issues.
+To keep a clean distribution, we recommend to work with a virtualenv:
+
+.. code-block:: bash
+
+    sudo apt update
```

Comment:
Here you are out of the distribution specific scope. Put the virtualenv package (if relevant) in each section dependencies and keep this distribution-agnostic.